real_decoder.go

```diff
@@ -290,6 +290,14 @@ func (rd *realDecoder) peek(offset, length int) (packetDecoder, error) {
 	return &realDecoder{raw: rd.raw[off : off+length]}, nil
 }
 
+func (rd *realDecoder) peekInt8(offset int) (int8, error) {
+	if rd.remaining() < offset+1 {
```

Comment:
What is `1` here? Would prefer a named constant instead of a magic number.

Reply:
It's the number of bytes in one byte. I can add a constant `BYTE_LEN`

Reply:
This file already has magic numbers for the lens of various types (`Int8`, `Int32`, etc). So this change fits in.

Reply:
yeah, I plan to fix that, but would love if you can change it to constant, so I have one less to fix :)
`const BYTE_LEN = 1` sounds great to me

Reply:
Golang naming conventions apparently prefer `byteLen`